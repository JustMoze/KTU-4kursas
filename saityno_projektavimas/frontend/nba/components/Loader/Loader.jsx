import React from 'react';
import ReactLoading from 'react-loading';
import colors from '../../styles/globalStyle.module.scss';


export default function Loader({size}) {
    return (
        <ReactLoading type="spin" color={colors.NBA_blue} height={size} width={size} />
    )
}

