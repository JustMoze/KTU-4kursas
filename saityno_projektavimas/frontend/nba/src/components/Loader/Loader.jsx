import React from 'react';
import ReactLoading from 'react-loading';


export default function Loader({size}) {
    return (
        <ReactLoading type="spin" color='#c80f2e' height={size} width={size} />
    )
}

