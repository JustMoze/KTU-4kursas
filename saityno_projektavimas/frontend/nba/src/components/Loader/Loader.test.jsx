import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loader from './Loader';


describe('Loader component', () => {
    const initProps = {
        size: 60
    } 
	it('Renders loader correctly', () => {
		let wrapper = shallow(<Loader {...initProps} />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
