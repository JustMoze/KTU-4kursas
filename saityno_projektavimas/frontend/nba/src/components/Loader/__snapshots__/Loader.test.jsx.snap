// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component Renders loader correctly 1`] = `
<Loading
  color="#c80f2e"
  delay={0}
  height={60}
  type="spin"
  width={60}
/>
`;
